komga-webui/src/locales/en.json

```diff
@@ -397,6 +397,7 @@
       "mixed": "MIXED",
       "tab_general": "General",
       "tab_tags": "Tags",
+      "tab_thumbnails": "Thumbnails",
```

Comment:
i was thinking we could align the naming with the one from Plex, which is also what we use in the doc (https://komga.org/guides/local-artwork-assets.html#poster-artwork), and use `Poster` instead of `Thumbnail`